share/completions/direnv.fish

```diff
@@ -0,0 +1,33 @@
+set -l commands allow permit grant block deny revoke edit exec fetchurl help hook prune reload status stdlib version
+
+set -l with_file allow permit grant block deny revoke edit
+
+complete -c direnv -f
+
+complete -c direnv -n "not __fish_seen_subcommand_from $commands" \
+    -a "allow permit grant" -d "Grants direnv permission to load the given .envrc or .env file"
+complete -c direnv -n "not __fish_seen_subcommand_from $commands" \
+    -a "block deny revoke" -d "Revokes the authorization of a given .envrc or .env file"
+complete -c direnv -n "not __fish_seen_subcommand_from $commands" \
+    -a edit -d "Opens PATH_TO_RC or the current .envrc or .env into an \$EDITOR and allow the file to be loaded afterwards"
+complete -c direnv -n "not __fish_seen_subcommand_from $commands" \
+    -a exec -d "Executes a command after loading the first .envrc or .env found in DIR"
+complete -c direnv -n "not __fish_seen_subcommand_from $commands" \
+    -a fetchurl -d "Fetches a given URL into direnv's CAS"
```

Comment:
What's "CAS" in this context?

Reply:
This wording is directly copied from `direnv --help` and even after looking at the source code I have no concrete answer to this. It probably stands for **C**ontent **A**ddressable **S**torage, but that's just an adjudicated guess.

Reply:
Tbh I'd suggest to upstream to rephrase that. Until then it's good enough for us.